components/automate-chef-io/content/docs/client-runs.md

```diff
@@ -216,4 +233,5 @@ curl -sSX POST "https://automate-url/api/v0/retention/nodes/delete-nodes/config"
 -H "api-token: $TOKEN"
 ```
 
-You will need an API token to send API requests. You can either [create a new token]({{< relref "api-tokens.md#creating-a-standard-api-token" >}}) or use a [Chef Automate 1 data collector token]({{< relref "data-collection.md#existing-data-collector-token-setup" >}}) if you are migrating from Chef Automate 1.
+You will need an API token to send API requests. 

```

Comment:
```suggestion
You will need an [API token]({{< relref "api-tokens.md#creating-api-tokens" >}}) to send API requests. 

```

we need to fix the link, but up to yall if you want to keep this a1 content.. seems like we shouldn't though?